src/CLITests/OutputGeneratorUnitTests.cs

```diff
@@ -56,12 +57,17 @@ private void VerifyAllMocks()
 
         private void SetOptions(VerbosityLevel verbosityLevel = VerbosityLevel.Default,
             string processName = null, int processId = -1,
-            string scanId = null)
+            string scanId = null, bool setScanId = true,
```

Comment:
This lets us leave the mock for ShowThirdPartyNotices unprogrammed in the couple of cases that need it that way